lib/zendesk_apps_support/location.rb

```diff
@@ -49,7 +49,17 @@ def self.all
       Location.new(id: 13, name: 'system_top_bar',
                    product_code: Product::STANDALONE_CHAT.code, v2_only: false),
       Location.new(id: 14, name: 'background',
-                   product_code: Product::CHAT.code)
+                   product_code: Product::CHAT.code),
+      Location.new(id: 15, name: 'top_bar', product_code: Product::SELL.code),
```

Comment:
Is Sell really building so many locations?

Reply:
@Mehul-Porwal We do have these in frontend. I was advised to add these typical ones (`nav_bar`, `top_bar` etc).

Reply:
@Mehul-Porwal we're launching with 4 location at start, but we'll be adding others listed sometime next year. If it's a relatively easy change to add new locations in the DB, we can start with 4 and take it from there.

Reply:
You do not need to add it to DB... just adding here is all that is required. Maybe just add the 4 you need at the moment and add more when required.

Reply:
I see, thanks @Mehul-Porwal. Why is the DB mentioned here though?

Reply:
@bbrzoska I asked whether I should add them to DB in other comment. It did not get attached to thread though :/

Reply:
The ids defined here are used in DB, when someone installs an app for 1Location.new(id: 17, name: 'deal_card', product_code: Product::SELL.code, collapsible: true),` we will use the code `17` in DB for the app_location, which will then we used to link to the definition here. Which means once in production, you can never change the id codes.

Reply:
Ah, that makes total sense. Thanks for the explanation @Mehul-Porwal!